data/sgid-index/index.html

```diff
@@ -3932,6 +3939,13 @@ <h6 id='show_filters' class='pointer'><i class="fas fa-search"></i> Show Filters
                 <td data-th="description" class="description">Riparian areas as defined by UGS</td>
                 <td data-th="service" class="service"><a href="https://opendata.gis.utah.gov/datasets/utah-riparian-areas" class="pull-right">{% include fa_icon.html api=true class="svg-inline--fa fa-w-20 fa-fw" %}</i></a></td>
             </tr>
+            <tr>
+                <td data-th="category" class="category">water</td>
+                <td data-th="name" class="name">Secondary Water Systems</td>
+                <td data-th="agency" class="agency">Water Resources</td>
+                <td data-th="description" class="description">This dataset attempts to represent the secondary water suppliers service area boundaries for the State of Utah. These are non-potable water suppliers for mostly lawn and garden use.</td>
+                <td data-th="service" class="service"></td>

```

Comment:
@ZachBeck this looks incomplete from the stewardship spreadsheet.

Reply:
stewardship has been updated

Reply:
ok this will run tonight and update and porter will give us an update next monday.